traffic_portal/app/src/common/modules/table/agGrid/CommonGridController.d.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export namespace CGC {
+    export enum OptionType {
+        Seperator = 0,
+        Button = 1,
+        Ancor = 2
+    }
+
+    export interface GridSettings {
+        rowClassRules?: any;
```

Comment:
I think this is a separate issue from rows being `any` - can this be typed more strictly?

Reply:
See above comment, this is "shadowing" an ag-grid type.